drivers/clock_control/Kconfig.stm32

```diff
@@ -252,6 +252,18 @@ config CLOCK_STM32_MCO1_SRC_PLLCLK
 	help
 	  Use PLLCLK as source of MCO1
 
+config CLOCK_STM32_MCO2_SRC_RC48
+	bool "RC48"
+	depends on SOC_SERIES_STM32H7X
+	help
+	  Use RC48 as source of MCO2
+
+config CLOCK_STM32_MCO2_SRC_PLL1Q
+	bool "PLL1Q"
```

Comment:
Could it be  'config CLOCK_STM32_MCO**2**_SRC_PLL1**P**' ? 
and  ' config CLOCK_STM32_MCO**1**_SRC_PLL**1Q**'

As from the ref man (RM0468) , MCO1 /MCO2 clock sources in the RCC clock configuration register (RCC_CFGR) are: 

**MCO1**[2:0]: Micro-controller clock output 1
```
000: HSI clock selected (hsi_ck) (default after reset)
001: LSE oscillator clock selected (lse_ck)
010: HSE clock selected (hse_ck)
011: PLL1 clock selected (pll1_q_ck)
100: HSI48 clock selected (hsi48_ck)

```
**MCO2**[2:0]: Micro-controller clock output 2
```
000: System clock selected (sys_ck) (default after reset)
001: PLL2 oscillator clock selected (pll2_p_ck)
010: HSE clock selected (hse_ck)
011: PLL1 clock selected (pll1_p_ck)
100: CSI clock selected (csi_ck)
101:LSI clock selected (lsi_ck)
```